src/transport/BLE.cpp

```diff
@@ -164,6 +165,9 @@ void BLE::OnBleConnectionComplete(void * appState, BLE_CONNECTION_OBJECT connObj
     CHIP_ERROR err = CHIP_NO_ERROR;
     BLE * ble      = reinterpret_cast<BLE *>(appState);
 
+    // We have already setup the ble endpoint for the same object, this should not happen, but we can tolerate it.
+    VerifyOrReturn(ble->mBleEndPoint == nullptr || !ble->mBleEndPoint->ConnectionObjectIs(connObj));
```

Comment:
Could we have an explanation on when this happens?

If it should never happen, we should probably print out a warning. This seems to silently ignore a state of 'this should not happen'

Reply:
Done, also created 4547 for final fix.